keras_nlp/models/backbone.py

```diff
@@ -141,6 +142,12 @@ def from_preset(
             config_overrides=kwargs,
         )
 
+    def save_to_preset(
+        self,
+        preset,
```

Comment:
Nit: remove the trailing comma, so this formats to one line like the other single arg methods in this file.

Reply:
Done!